### DISCUSSION TITLE
Global Adress validation mockups

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#1231] [Office hours] [answered]
Hi team, this are the highlights I wanted to go through on OH for the [Global Address Validation project](https://www.figma.com/file/sIZF6N5J08uaPoGB96WAjG/PN-GAV-Global-Address-Validatiomn?node-id=0%3A1):
**Disclosures:** Is that the correct use? 
**[Alert on ‘create and Address’:](https://www.figma.com/proto/sIZF6N5J08uaPoGB96WAjG/PN-GAV-Global-Address-Validatiomn?node-id=35%3A5&viewport=-3040%2C2629%2C0.18070173263549805&scaling=min-zoom)** Is the a correct placement for the alert that appears when the users interacts with that disclosure?
**Modal alignment:** I tried to align content to the spacing established for the modal component, but I find it too wide/high (it says that is 50px, but if you measure the example it has a spacing of 17px). 
**‘Cancel’ button:** When user press this button all information will not be saved. Should this button have a sort of alert? 
**[Managing tooltip in a table's header](https://www.figma.com/proto/sIZF6N5J08uaPoGB96WAjG/PN-GAV-Global-Address-Validatiomn?node-id=1%3A45746&viewport=-3040%2C2629%2C0.18070173263549805&scaling=min-zoom)**: It is the right use for a tooltip?
**[Edit Address button:](https://www.figma.com/proto/sIZF6N5J08uaPoGB96WAjG/PN-GAV-Global-Address-Validatiomn?node-id=35%3A5&viewport=-2239%2C2333%2C0.13111110031604767&scaling=min-zoom)** I want to know if this is the best option for this interaction or if I should consider adding it to the footer.

**Comment by SiTaggart** [comment]
Hi @lgalloherran 

Thanks for posting these so early, it gives the team so much time to review before your Office hours session. 🙌 

We'll review and post comments

**Comment by kate-porter** [comment]
Hey @lgalloherran - just wanted to followup on the question regarding modal alignment.

I think what you're referring to are the spacing helpers on the modal, which are labeled as `space-50`, like so:
![image](https://user-images.githubusercontent.com/5374045/110009005-a3a24400-7cd9-11eb-87ef-4fba85e0e69e.png)

What this is actually referring to is the name of the spacing token from Paste that we are using for the modal padding. The name of the token is `space-50`, but the value under the hood is 16px. Check out the spacing tokens listed here: https://paste.twilio.design/tokens/#spacings

Hopefully that helps. Let me know if you have any more questions.

**Reply by lgalloherran** [reply]
Thanks! @katieporter  for clarifying this, I'll check the equivalence then.

**Comment by SiTaggart** [comment, marked as answer]
Link to the recording can be found here https://www.loom.com/share/8b6a519fd2a344dfb94dc966e59a18e6# (pw is in [#help-design-system](https://www.loom.com/share/8b6a519fd2a344dfb94dc966e59a18e6#) slack)

In the discussion we covered:

- That potentially the use of disclosures might make validating two distinct pieces of data on the same step tricky, and maybe explore the second disclosure being a sub-step so it's more focused
- That inputs can be placed in multi column grids
- Use of a success toast was perfect
